aws/sentry/aurora-postgres.tf

```diff
@@ -0,0 +1,204 @@
+  count   = local.postgres_cluster_enabled ? 1 : 0
+  length  = 26
+  special = true
+}
+
+locals {
+  # Some of these locals used to be computed. Keep them as local just for consistency,
+  # even though now they are just copies of variables.
+  postgres_cluster_enabled = var.postgres_cluster_enabled
+  postgres_db_name         = var.postgres_db_name
+  postgres_admin_user      = length(var.postgres_admin_user) > 0 ? var.postgres_admin_user : "sentry"
+  postgres_admin_password  = length(var.postgres_admin_password) > 0 ? var.postgres_admin_password : join("", random_string.postgres_admin_password.*.result)
+}
+
+module "aurora_postgres" {
+  source           = "git::https://github.com/cloudposse/terraform-aws-rds-cluster.git?ref=tags/0.16.0"
+  namespace        = var.namespace
+  stage            = var.stage
+  name             = var.postgres_name
+  attributes       = ["sentry"]
+  engine           = "aurora-postgresql"
+  cluster_family   = "aurora-postgresql9.6"
+  engine_version   = var.aurora_postgres_engine_version
+  instance_type    = var.postgres_instance_type
+  cluster_size     = var.postgres_cluster_size
+  admin_user       = local.postgres_admin_user
+  admin_password   = local.postgres_admin_password
+  db_name          = local.postgres_db_name
+  db_port          = "5432"
+  vpc_id           = module.kops_metadata.vpc_id
+  subnets          = module.kops_metadata.private_subnet_ids
+  zone_id          = local.zone_id
+  cluster_dns_name = "master.postgres-sentry"
+  reader_dns_name  = "replicas.postgres-sentry"
+  security_groups  = [module.kops_metadata.nodes_security_group_id]
+  enabled          = var.postgres_cluster_enabled
+
+  iam_database_authentication_enabled = var.postgres_iam_database_authentication_enabled
+}
+
+resource "aws_ssm_parameter" "aurora_postgres_database_name" {
+  count       = local.postgres_cluster_enabled ? 1 : 0
+  name        = format(local.chamber_parameter_format, local.chamber_service, "sentry_postgres_database")
+  value       = module.aurora_postgres.database_name
+  description = "Aurora Postgres Database Name for Sentry"
+  type        = "String"
+  overwrite   = "true"
+}
+
+resource "aws_ssm_parameter" "aurora_postgres_master_username" {
+  count       = local.postgres_cluster_enabled ? 1 : 0
+  name        = format(local.chamber_parameter_format, local.chamber_service, "sentry_postgres_user")
+  value       = module.aurora_postgres.master_username
+  description = "Aurora Postgres Username for Sentry's master DB user"
+  type        = "String"
+  overwrite   = "true"
+}
+
+resource "aws_ssm_parameter" "aurora_postgres_master_password" {
+  count       = local.postgres_cluster_enabled ? 1 : 0
+  name        = format(local.chamber_parameter_format, local.chamber_service, "sentry_postgres_password")
+  value       = local.postgres_admin_password
+  description = "Aurora Postgres Password for Sentry's master DB user"
+  type        = "String"
```

Comment:
```suggestion
  type        = "SecureString"
```

Reply:
We generally do not use SecureString because then you have to specify a KMS key but it doesn't really add any security because anyone who can read `/kops` has access to the chamber KMS key and the Terraform state file where the key is unencrypted.

Reply:
ok agree